hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -567,22 +577,30 @@ def check_all_users_home_directory(max_system_uid):
     return True if error == [] else str(error)
 
 
-def check_users_home_directory_permissions(reason=''):
+def check_users_home_directory_permissions( non_login_shell='/sbin/nologin', max_allowed_permission='750', except_for_users='' ):
```

Comment:
The value of non_login shell is '/sbin/nologin' in other operating systems. But in ubuntu16, the path of non login shell is '/usr/sbin/nologin'.
So, I now we are taking the path of non_login_shell as an argument. It's default value is still '/sbin/nologin', so it won't affect existing profiles.